### ISSUE TITLE
bytes.decimal codecs all use the same scale and precision

### ISSUE THREAD

**Issue by henry-megarry** [linkedin/goavro#193]
will only use the first bytes.decimal scale and precision that it finds making later fields the wrong value. In the example below field2 should be "13/20" or ".65" but it uses the scale of field1 so it is also "65/1"

when using schema2(which is the same schema but have field2 defined first) then both field1 and field2 are both the "13/20"

https://play.golang.org/p/_OnGdYexUIA

**Comment by sammythebull23** [comment]
I'm running across same exact issue. @henry-megarry you figure out any way to work around this?

**Comment by sammythebull23** [comment]
It looks like this will be resolved in PR: #195 excellent.

**Comment by mooseburgr** [comment]
For what it's worth, I ran your same playground test locally with the fix in #238 present and got output:
```
field1 with schema1: 65/1
field2 with schema1: 13/20
field1 with schema2: 65/1
field2 with schema2: 13/20
```
